sites/friday/src/add-services/clickhouse.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Clickhouse"
+weight: -110
+description: 
+---
+
+Clickhouse is a high-performance column-oriented, distributed, OLAP (Online Analytical Processing) database.</br>
+It is open source and allows you to generate real-time analytical data reports using SQL queries.</br>
```

Comment:
I would remove `is open source and`. All our services are open source and we never specify it.